src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -201,13 +200,6 @@ const useCoreChatSystemHook = () => {
       websocketData?.originalMembers?.forEach(async (member) => {
         const userMember = UserSchema.fromMemberWebsocketObject(member, websocketData?.channel?.id);
         await userMember.saveOrUpdateIfExists(localDb);
-
-        const memberSchema = ChannelListMemberSchema.fromWebsocketObject(
-          websocketData?.channel?.id,
-          websocketData?.message?.id,
-          member
-        );
-        await memberSchema.save(localDb);
       });
     } catch (e) {
       console.log('error on memberSchema');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the new hunk, the creation and saving of `memberSchema` has been removed. This might lead to data inconsistency if other parts of the application rely on this schema being updated here. Ensure that this change doesn't affect other parts of the system that might be relying on `ChannelListMemberSchema`.
```diff
-        const memberSchema = ChannelListMemberSchema.fromWebsocketObject(
-          websocketData?.channel?.id,
-          websocketData?.message?.id,
-          member
-        );
-        await memberSchema.save(localDb);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->